### PR TITLE
Add jd45 via keymap

### PR DESCRIPTION
**Pull request by matthewdias**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

**Comment by wilba** [comment]
Maybe I should move jd45 and jd40 into wilba_tech since they are PCBs I designed, and give them VID/PIDs to match?
What do QMK maintainers think?

**Comment by drashna** [comment]
> Maybe I should move jd45 and jd40 into wilba_tech since they are PCBs I designed, and give them VID/PIDs to match?
> What do QMK maintainers think?

Sounds fine to me.  Though, that would mean it needs to target the develop branch.

**Review by drashna — APPROVED** [review]

**Comment by Erovia** [comment]
> Maybe I should move jd45 and jd40 into wilba_tech since they are PCBs I designed, and give them VID/PIDs to match?
> What do QMK maintainers think?

We can merge this as is and Wilba's move/pid PR later (going through `develop` with the latter)
Alternatively, this PR could incorporate the move and VID:PID change as well and be retargeted to the `develop` branch. This would probably cause less disruption as the VID:PID will not change after the `via` keymap is merged.

It's up to you @matthewdias and @wilba.

**Comment by matthewdias** [comment]
The latter sounds more ideal to me. Happy to resubmit after the move

**Comment by wilba** [comment]
@matthewdias OK I will do the move inc. adding VIA support and start a new PR against `develop`. Start a PR to https://github.com/the-via/keyboards with the keyboard definition you have already.

File: keyboards/jd45/keymaps/via/rules.mk
Comment (fauxpark) [inline keyboards/jd45/keymaps/via/rules.mk:2]:
```suggestion
```
Technically this isn't needed as `rules.mk` stuff is case sensitive, and the `MIDI_ENABLE` at the keyboard level is `YES` instead of `yes`.
Might as well remove that too in this PR.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
